src/f3/mod.rs

```diff
@@ -100,6 +113,32 @@ pub fn run_f3_sidecar_if_enabled(
     }
 }
 
+pub fn import_f3_snapshot(
+    chain_config: &ChainConfig,
+    _rpc_endpoint: String,
+    _f3_root: String,
+    _snapshot: String,
```

Comment:
nit: `_` prefix is kind of deceiving here - it is actually used, but not on all features. I feel more expressive would be to explicitly allow unused params.

Reply:
Fixed.